datalake/src/main/java/com/sequenceiq/datalake/repository/SdxStatusRepository.java

```diff
@@ -20,4 +22,10 @@ public interface SdxStatusRepository extends CrudRepository<SdxStatusEntity, Lon
     List<SdxStatusEntity> findDistinctFirstByStatusInAndDatalakeIdInOrderByIdDesc(Collection<DatalakeStatusEnum> datalakeStatusEnums,
```

Comment:
Do you think we can delete this?

Reply:
This is the rename. The other method is deleted, but it is not used anywhere.